src/nvidia_rag/utils/vdb/elasticsearch/es_queries.py

```
@@ -189,3 +189,43 @@ def get_delete_document_info_query_by_collection_name(collection_name: str):
     }
     return query_delete_document_info
+
+def get_weighted_hybrid_custom_query(
+    embedding_model,
+    dense_weight,
+    sparse_weight,
+    k: int,
+    num_candidates: int = 100,
+):  
+    
+    def weighted_hybrid_query_builder(query_body: dict, query_text: str):
```

Ruff (0.14.11)

Unused function argument: `query_body`

(ARG001)